### PR TITLE
Updates to the preseq plotting and snpEff plotting scripts

### PR DESCRIPTION
**Pull request by ewels**

### CODE REVIEW

File: ngi_visualizations/preseq_complexity_curves/plot_complexity_curves.py [hunk 1]
```diff
@@ -25,10 +25,19 @@ def plot_complexity_curves(ccurves, real_counts_path=False, output_name='complex
...
     """
-    
+
     if x_min < 0 or x_max <= x_min:
         sys.exit("problem with x-min or x-max ({}, {}). x-min must be equal or higher to 0 and less than x-max".format(x_min, x_max))
-    
+
+    # Covert limit counts to coverage
+    if coverage > 0:
+        if read_length == 0:
```
Comment (vezzi) [inline ngi_visualizations/preseq_complexity_curves/plot_complexity_curves.py]: You should be able to integrate this logic whit ArgumentParser, not  a big thing but makes code cleaner... sorry for not coming back to you @ewels :-D

File: ngi_visualizations/preseq_complexity_curves/plot_complexity_curves.py [hunk 2]
```diff
@@ -193,22 +238,25 @@ def computeLimit(value, ccurve_TOTAL_READS):
...
         help="File name for file with three columns - BAM filename, total number reads, number of unique reads (space delimited)")
-    parser.add_argument('-o', '--output-name', dest='output_name', type=str, default='complexity_curves', 
+    parser.add_argument('-o', '--output-name', dest='output_name', type=str, default='complexity_curves',
         help="Output name (.png will be automatically added)")
-    parser.add_argument('-m', '--x-min', dest='x_min', type=int, default=0, 
+    parser.add_argument('-m', '--x-min', dest='x_min', type=int, default=0,
         help="Lower x-limit (default 0)")
-    parser.add_argument('-x', '--x-max', dest='x_max', type=int, default=500000000, 
+    parser.add_argument('-x', '--x-max', dest='x_max', type=int, default=500000000,
         help="Upper x-limit (default 500 million)")
```
Comment (vezzi) [inline ngi_visualizations/preseq_complexity_curves/plot_complexity_curves.py]: Maybe if coverage > 0 you should set --x-max default value to 60X

Comment (ewels) [inline ngi_visualizations/preseq_complexity_curves/plot_complexity_curves.py]: Could do, or just make the default have the number of reads that corresponds to 60X for human..

Comment (vezzi) [inline ngi_visualizations/preseq_complexity_curves/plot_complexity_curves.py]: yep however this does not scale to other genomes

**Comment by vezzi** [comment]
cool for me, I did this review as @ewels has been waiting for me for 2 month or more!!!
if you want to implement the default coverage tell me otherwise I will merge

**Comment by ewels** [comment]
I can't really be bothered just now to be honest :) I agree that it would be nice to improve it so that the argument parser handles more of the logic though. One day...

**Pull request merged**